src/data/sym/basic.lean

```diff
@@ -28,7 +28,9 @@ symmetric powers
 
 -/
 
-universes u
+open function
```

Comment:
Please split the changes in this file to a new PR.

Reply:
I've moved all the changes to `data/sym/basic` into #12479